docker/Dockerfile

```diff
@@ -30,7 +29,8 @@ RUN yum install -y \
     micromamba clean --yes --all && \
     echo 'export  PATH="/opt/micromamba/bin:${PATH}"' >> /root/.bashrc && \
```

Comment:
You can just write everything to /etc/profile rather than to '/root/.bashrc` - that means those lines might be executed twice.

In general when trying to write dockerfiles, I try to make sure $HOME (so `/root`) is completely empty when the image is done building. This prevents other accidental issues that cause things to work as root but not elsewhere. In repo2docker we actually temporarily unset $HOME to prevent this: https://github.com/jupyterhub/repo2docker/blob/main/repo2docker/buildpacks/conda/install-base-env.bash#L15